### DISCUSSION TITLE
Scrolling line for Data Points

### DISCUSSION THREAD

**Discussion by morton027** [kalkih/mini-graph-card#1008] [Q&A]
Hi

Anyone know if this is possible on the graphs, to add a vertical scrolling line to view data points from the graph?

On iPhone and iPad it's nigh on impossible to select individual points.

**Comment by ildar170975** [comment]
> to add a vertical scrolling line to view data points from the graph?

Unclear.
Please provide a picture explaining your idea.

**Comment by morton027** [comment]
As a rough example, when you hover over the graph area the vertical line would appear and drag along the graph, showing the data from said point in time.

Finding the points on the line can be difficult on the mobile app and even on a tablet.

Thanks

Nick Morton

**Reply by ildar170975** [reply]
I asked for a detailed picture describing your FR. Let's say that a "picture" = "image".
What you wrote are some ideas - not enough even for a "draft of a technical assignment".
